nodejs/week1/homework/warmup.js

```diff
@@ -0,0 +1,23 @@
+console.log("inside warmup file");
+
+//Circle calculator
+
+class Circle {
+  constructor(radius) {
+    this.radius = radius;
+  }
+  getDiameter() {
+    return this.radius * 2;
+  }
+  getCircumference() {
+    return this.radius * 2 * Math.PI;
+  }
+  getArea() {
+    return Math.PI * this.radius ** 2;
+  }
+}
+
```

Comment:
Clean, clear, correct. 👌🏽  👏🏽  ✨ 

Consideration for improvements:

- We are getting some crazy fractions for circumference and area. Perhaps would make sense to round the numbers? 
- `Math.PI * this.radius ** 2` is perfectly fine, but for readability and practice `Math.pow()` could be tried out to perform one of the operations here.